### DISCUSSION TITLE
When can we expect the official helm charts to be released?

### DISCUSSION THREAD

**Discussion by shikha-priyedarshi** [uber/cadence#6024] [General]
When can we expect the official helm charts to be released?

**Comment by demirkayaender** [comment]
This is expected to be released around the end of July.

**Reply by tamalsaha** [reply]
Any update?

**Reply by demirkayaender** [reply]
We have a working version ready, and there are a couple of polishing pieces left. I think releasing this will take about a month.

**Reply by tamalsaha** [reply]
If you don't sharing it publicly before it is fully baked, that will be greatly appreciated. I am learning how to use Cadence. The helm chart will be handy to get going. Thanks.

**Reply by mantas-sidlauskas** [reply]
adding here for visibility: https://github.com/uber/cadence-charts
